mslib/plot.py

```diff
@@ -87,84 +93,55 @@ def __init__(self):
         self.params["basemap"].update(self.config["predefined_map_sections"][section]["map"])
         print(self.params)
         self.bbox_units = self.params["bbox"]
-        self.wps = load_from_ftml(filename)
+        self.wpslist = []
+        self.wpslist = load_from_ftml(filename)
+        self.wps = self.wpslist[0]
+        self.wp_model_data = self.wpslist[1]
         self.wp_lats, self.wp_lons, self.wp_locs = [[x[i] for x in self.wps] for i in [0, 1, 3]]
-        self.wp_presss = [mslib.utils.thermolib.flightlevel2pressure(wp[2] * units.hft).to("Pa").m for wp in self.wps]
+        self.wp_press = [mslib.utils.thermolib.flightlevel2pressure(wp[2] * units.hft).to("Pa").m for wp in self.wps]
         self.fig.clear()
         self.ax = self.fig.add_subplot(111, zorder=99)
         self.path = [(wp[0], wp[1], datetime.datetime.now()) for wp in self.wps]
+        self.vertices = [list(a) for a in (zip(self.wp_lons, self.wp_lats))]
```

Comment:
This should be removed. The "vertices" of the msui classes are often in different units then lon/lat. The restructuring of using update_from_waypoints should have made this superfluous anyway.